draft-ietf-rtcweb-jsep.xml

```diff
@@ -1833,12 +1731,18 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             appropriate for the media type, as specified in
             <xref target="I-D.ietf-rtcweb-security-arch" /></t>
           </list></t>
+
+          <t>The SDES SRTP keying mechanism from <xref
+          target="RFC4568"/> MUST NOT be implemented, as discussed in
+          <xref target="I-D.ietf-rtcweb-security-arch"/>.</t>
+
         </section>
         <section title="Profile Names and Interoperability"
         anchor="sec.profile-names">
 
           <t>For media m= sections, JSEP implementations MUST support both
-          the "UDP/TLS/ RTP/SAVPF" and "TCP/DTLS/RTP/SAVPF" profiles
+          the "UDP/TLS/RTP/SAVPF" and "TCP/DTLS/RTP/SAVPF" profiles,
```

Comment:
I think we have changed our position on this and now prefer always using UDP/XXXX.

    o  To properly indicate use of DTLS, the <proto> field MUST be set to
        "UDP/TLS/RTP/SAVPF", as specified in [RFC5764], Section 8.

@ekr, does this match your understanding?